actionmailer/lib/action_mailer/base.rb

```diff
@@ -163,7 +163,7 @@ module ActionMailer
   #
   # Multipart messages can also be used implicitly because Action Mailer will automatically detect and use
   # multipart templates, where each template is named after the name of the action, followed by the content
-  # type. Each such detected template will be added as a separate part to the message.
+  # type. Each detected template will be added to the message, as a separate part.
```

Comment:
OK, thanks @vipulnsward. Do you want me to revert the whole line? Or change it to:
`Each such detected template will be added to the message, as a separate part.`

I personally think 'such' is unnecessary but happy to put it back in if you'd like